docs/iris/src/userguide/real_and_lazy_data.rst

```diff
@@ -0,0 +1,246 @@
+.. _real_and_lazy_data:
+
+==================
+Real and Lazy Data
+==================
+
+What is Real and Lazy Data?
+---------------------------
+
+Every Iris cube contains an n-dimensional data array, which could be real or
+lazy.
+
+Real data is contained in a NumPy array which has a shape, a data type, some
+other useful information and many data points, each of which use up a small
+allocation of memory.
+
+Lazy data is contained in a conceptual array which retains the information
+about its real counterpart but has no actual data points, so its memory
+allocation is much smaller.  This will be in the form of a Dask array.
```

Comment:
@corinnebosley Perhaps format `Dask` as a link to the dask readthedocs, so the user can follow this if they want ...